### PR TITLE
Cleanup to the getting started guide.

### PR DESCRIPTION
**Pull request by geertj**
The image styles are currently a crude hack. Someone with CSS experience should
have a look at this (Eric that may be you :).

The main changes are:

* Don't explain how to get started with the three options. Assume people that select an option already have Vagrant, Docker, or Google.

* Focus Vagrant on Windows and Mac users, and Docker on Linux users. I expect this to be the main audiences for these options.

* Since we have separate chapters on configuration and adding a provider, there should be no need to repeat those in the Vagrant, Docker and Cloud sections.

* Added some instructions for port mapping on Vagrant. I've chosen port 8443 as the standard port, and use that for Docker as well.

### CODE REVIEW

File: site/docs/get-started/cloud.md [hunk 1]
```diff
@@ -1,27 +1,30 @@
 ---
 layout: doc_page
-doc_group: start
 title: Easy Install Using A Public Cloud Provider
 ---
-The installation of ManageIQ consists on two phases:
 
- 1. Install ManageIQ as a cloud image in the cloud
- 2. Connect it to a managed system, and start working with it.
+You can try ManageIQ in one of the public clouds that are suppored. The
```
Comment (sergio-ocon) [inline site/docs/get-started/cloud.md]: supported

**Comment by sergio-ocon** [comment]
LGTM

**Comment by hayesr** [comment]
@sergio-ocon @geertj This needs to be rebased.

**Comment by geertj** [comment]
@hayesr rebased, can you please merge?

I added some very hacky CSS for the screenshots. This probably needs to be redone by someone who actually understands this stuff..

**Pull request merged**